apps/wallet-mock/src/support/event-trigger.ts

```diff
@@ -0,0 +1,58 @@
+// For now it supports only simple transaction review event, ideally it should be extended to support all possible workflows
+
+Cypress.Commands.add('triggerTransactionEvent', (event, data) => {
+  cy.window().then((win) => {
+    win.document.body.dispatchEvent(
+      new CustomEvent('new_interaction', {
+        detail: {
+          name: 'INTERACTION_SESSION_BEGAN',
+          data: { workflow: 'TRANSACTION_REVIEW' },
+          traceID: '1',
+        },
+      })
+    )
+  })
```

Comment:
Only needs to be done before a single workflow not every single time

Reply:
Make sense. I will divide into 3 separate cypress functions